app/src/main/java/com/yeonae/chamelezone/data/source/remote/place/PlaceRemoteDataSourceImpl.kt

```diff
@@ -329,33 +310,99 @@ class PlaceRemoteDataSourceImpl private constructor(private val placeApi: PlaceA
+            imageNumbers.add(
+                RequestBody.create(
+                    MediaType.parse("text/plain"), imageNumber[i].toString()
+                )
+            )
+        }
+
         placeService.updatePlace(
+            placeNumber,
             imageList,
             memberNumber,
-            keyword,
-            name,
             address,
-            openingTime,
             phoneNumber,
             content,
             latitude,
-            longitude
-        ).enqueue(object :
-            Callback<ResponseBody> {
-            override fun onResponse(
-                call: Call<ResponseBody>,
-                response: Response<ResponseBody>
-            ) {
+            longitude,
+            imageNumbers
+        ).enqueue(object : Callback<ResponseBody> {
+            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
+                Log.e("tag", t.toString())
+            }
+
+            override fun onResponse(call: Call<ResponseBody>, response: Response<ResponseBody>) {
                 if (response.code() == Network.SUCCESS) {
                     callBack.onSuccess(true)
                 }
             }
 
-            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
-                Log.e("tag", t.toString())
-            }
         })
+    }
+
+    override fun updateKeyword(
+        placeNumber: Int,
+        keywordNames: List<Int>,
+        placeKeywordNumber: List<Int>,
+        callBack: PlaceCallBack<Boolean>
+    ) {
+        val jsonObject = JsonObject().apply {
+            addProperty("keywordName", keywordNames.toString())
+            addProperty("placeKeywordNumber", placeKeywordNumber.toString())
+        }
+        placeService.updateKeyword(placeNumber, jsonObject)
+            .enqueue(object : Callback<ResponseBody> {
+                override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
+                    Log.e("tag", t.toString())
+                    Log.d("step5", call.request().body()?.let { bodyToString(it) })
+                }
+
+                override fun onResponse(
+                    call: Call<ResponseBody>,
+                    response: Response<ResponseBody>
+                ) {
+                    Log.d("responseCode", response.code().toString())
+                    Log.d("step5", call.request().body()?.let { bodyToString(it) })
+                    if (response.code() == Network.SUCCESS) {
+                        callBack.onSuccess(true)
+                    }
+                    if (response.code() == Network.SUCCESS) {
+                        callBack.onSuccess(true)
+                    }
+                }
 
+            })
+    }
+
+    override fun updateOpeningHours(
+        placeNumber: Int,
+        openingTimes: List<String>,
+        callBack: PlaceCallBack<Boolean>
+    ) {
+        val jsonObject = JsonObject().apply {
+            addProperty("openingTime", openingTimes.toString())
```

Comment:
```suggestion
            addProperty("openingTime", openingTimes.toString())
```
이렇게하면 서버에서 받을때 [1, 2, 3, 4]가 아니라 "[1, 2, 3, 4]" 이렇게 들어가지 않나요?
이렇다면 그에따른 이슈는 없나요?

Reply:
수정했습니다
d2a8582